qlty-plugins/plugins/linters/checkstyle/plugin.toml

```diff
@@ -0,0 +1,21 @@
+config_version = "0"
+
+[plugins.definitions.checkstyle]
+runnable_archive_url = "https://github.com/checkstyle/checkstyle/releases/download/checkstyle-${version}/checkstyle-${version}-all.jar"
+runtime = "java"
+file_types = ["java"]
+latest_version = "10.21.4"
+known_good_version = "10.21.4"
+version_command = "java -jar ${linter}/checkstyle --version"
+description = "Java linter"
+config_files = ["checker.xml"]
+
+[plugins.definitions.checkstyle.drivers.lint-java]
```

Comment:
Does checkstyle have a formatter / rewrite mode or only lint?

Reply:
Nothing of the sort in the docs https://checkstyle.org/cmdline.html